mlir/docs/BufferDeallocationInternals.md

```diff
@@ -664,8 +664,6 @@ merged into a single step. Canonicalization removes the clone operation and
 func.func @reuseTarget(%arg0: memref<2xf32>, %result: memref<2xf32>){
   %temp = memref.alloc() : memref<2xf32>
   test.generic {
-    args_in = 1 : i64,
-    args_out = 1 : i64,
```

Comment:
Just deleting will still leave IR incorrect. The IR below should be like (Notice the ins , outs and math.exp, and braces ...

```
    %alloc = memref.alloc() : memref<2xf32>
    linalg.generic {indexing_maps = [#map, #map], iterator_types = ["parallel"]} ins(%arg1 : memref<2xf32>) outs(%alloc : memref<2xf32>) {
    ^bb0(%in: f32, %out: f32):
      %1 = math.exp %in : f32
      linalg.yield %1 : f32
    }
``

Reply:
Thanks for spotting that. This example is broken in other ways too, e.g (the parser rejects this):
```mlir
%result = bufferization.clone %temp : (memref<2xf32>) -> (memref<2xf32>)
```

Also, at the top of the file:
```
**Note:** This pass is deprecated. Please use the ownership-based buffer
deallocation pass instead.
```

I feel that I should skip this file altogether, it clearly hasn't been maintained for a while now.

Reply:
Even if you don’t fix everything … fixing some parts is already an improvement .

Reply:
What are you suggesting? Partial update in this file? TBH, I'd rather skip it.

Reply:
Up to you really.
 I meant if args_in args_out are present in files like `mlir/include/mlir/Dialect/Bufferization/Transforms/Passes.td` which lot of folks read, removing  it is still better than 'fix everything or fix nothing'
then

Reply:
Sounds like I've addressed your comment then? Asking as according to GitHub you are still "requesting changes" :)